templates/docs/patterns/equal-height-row.md

```diff
@@ -0,0 +1,88 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Equal height row | Components
+---
+
+The equal height row component aims to provide consistent alignment for grid items within a row format. This is achieved using the CSS `subgrid` feature which allows column grids to share the same layout used in the parent row grid.
+
+In addition to enforcing column grid items alignment within each row, responsive behaviour is provided out of the box as shown below:
+
+<table>
+  <thead>
+    <tr>
+      <th style="width: 50ch">Screen size (px)</th>
+      <th>Behaviour</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Less than <code>$breakpoint-small</code></td>
+      <td>Each column spans across the entire parent grid. Column items are vertically stacked.</td>
+    </tr>
+    <tr>
+      <td><code>$breakpoint-small</code> - <code>$breakpoint-large</code></td>
+      <td>Each column spans across the entire parent grid. The first item within each column is placed on the left of the other column items.</td>
+    </tr>
+    <tr>
+      <td>Greater than <code>$breakpoint-large</code></td>
+      <td>Columns within the row are displayed horizontally. Column items are vertically stacked.</td>
+    </tr>
+  </tbody>
+</table>
+
+This component is an extension of the [grid component](/docs/patterns/grid). There are several usage variations detailed in the examples below.
+
+## Four items per column and full-screen dividers
```

Comment:
To make it feel less like an exception let's simplify this title to "Full example" maybe

```suggestion
## Full example
```